### PR TITLE
office hours 2020-04-14

### PR DESCRIPTION
**Pull request by directionless**

### CODE REVIEW

File: docs/office-hours/20200414_meeting_notes.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# osquery office hours 2019-04-14
+
+YouTube Link:
```
Comment (directionless) [inline docs/office-hours/20200414_meeting_notes.md:3]:
```suggestion
YouTube Link: https://www.youtube.com/watch?v=JlV3YkJ5rDI
```

File: docs/office-hours/20200414_meeting_notes.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+This highlights a bit of a gap in how we document some of our policies
+and goals about what kind of code belongs in osquery.
+
+## Build system without Buck
+
+When `osql` first brought cmake in, many compromises were made to be
+compatible with Buck. Now that we have removed Buck, the cmake can be
+simpler.
+
+Some discussion about how this translates to depenancies. It's
```
Comment (mike-myers-tob) [inline docs/office-hours/20200414_meeting_notes.md:74]: `dependencies`

File: docs/office-hours/20200414_meeting_notes.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+positive to have explicit defines for each target. But, that makes it
+hard to use an in IDEs. Also, the folder structure isn't on disk,
+which makes using external tools harder.
+
+One thing we can do now, is to start refacoring names to be
```
Comment (mike-myers-tob) [inline docs/office-hours/20200414_meeting_notes.md:79]: `refactoring`

File: docs/office-hours/20200414_meeting_notes.md (hunk 2 above)
Comment (directionless) [inline docs/office-hours/20200414_meeting_notes.md:74]:
```suggestion
Some discussion about how this translates to dependencies. It's
```

File: docs/office-hours/20200414_meeting_notes.md (hunk 3 above)
Comment (directionless) [inline docs/office-hours/20200414_meeting_notes.md:79]:
```suggestion
One thing we can do now, is to start refactoring names to be
```

File: docs/office-hours/20200414_meeting_notes.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+## Any PRs people want to discuss
+
+### Add status and statements to distributed query results
+https://github.com/osquery/osquery/pull/6352
+
+This came up last week. The PR includes both statuses and statements,
+and there's been some discussion about whether the statements are
+needed.
+
+Some discussion about this, is that there's general hesitency. This is
```
Comment (mike-myers-tob) [inline docs/office-hours/20200414_meeting_notes.md:16]: hesitancy

File: docs/office-hours/20200414_meeting_notes.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+simpler.
+
+Some discussion about how this translates to dependencies. It's
+positive to have explicit defines for each target. But, that makes it
+hard to use an in IDEs. Also, the folder structure isn't on disk,
+which makes using external tools harder.
+
+One thing we can do now, is to start refactoring names to be
+shorter. This should be able to happen aside from any include
+organziational questions.
```
Comment (mike-myers-tob) [inline docs/office-hours/20200414_meeting_notes.md:81]: organizational

File: docs/office-hours/20200414_meeting_notes.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+## Build system without Buck
+
+When `osql` first brought cmake in, many compromises were made to be
+compatible with Buck. Now that we have removed Buck, the cmake can be
+simpler.
+
+Some discussion about how this translates to dependencies. It's
+positive to have explicit defines for each target. But, that makes it
+hard to use an in IDEs. Also, the folder structure isn't on disk,
```
Comment (mike-myers-tob) [inline docs/office-hours/20200414_meeting_notes.md:76]: in IDEs

**Pull request merged**